src/com_tjvendors/admin/models/vendorfee.php

```diff
@@ -171,9 +171,8 @@ public function save($data)
 					$data['id'] = (int) $this->getState($this->getName() . '.id');
 				}
 
-				$dispatcher = JDispatcher::getInstance();
 				PluginHelper::importPlugin('tjvendors');
-				$dispatcher->trigger('tjVendorsOnAfterVendorFeeSave', array($data, $isNew));
+				Factory::getApplication()->triggerEvent('tjVendorsOnAfterVendorFeeSave', array($data, $isNew));
```

Comment:
dont triggers needs to be starting as `on`

Reply:
done